tests/ccgbugs/mabi_check.nim

```diff
@@ -0,0 +1,8 @@
+type GoodImportcType {.importc: "signed char", nodecl.} = char
+  # "good" in sense the sizeof will match
+type BadImportcType {.importc: "unsigned char", nodecl.} = uint64
+  # "sizeof" check will fail
+
+## ensures cgen
+discard GoodImportcType.default
+discard BadImportcType.default
```

Comment:
fixed differently in a way that can't be optimized away; in any case the tests would fail if it ever were optimized away, so no bugs could creep.
note that `echo` would force to use a separate header files as described here https://github.com/timotheecour/Nim/issues/65#issuecomment-611803212